frontend/src/components/pages/CreateReview/BookModal.tsx

```diff
@@ -0,0 +1,374 @@
+      isbn !== "" &&
+      price > 0 &&
+      coverImage !== "" &&
+      genre !== "" &&
+      minAge > 0 &&
+      maxAge > 0 &&
+      maxAge >= minAge &&
+      isEmptyOrValidISBN()
+    );
+  };
+
+  /** Creates and saves a new book object, then closes the modal */
+  const updateBookObj = () => {
+    const authorObjs: Author[] = [];
+
+    authors.forEach((author) => {
+      authorObjs.push({
+        fullName: author,
+        displayName: author,
+        attribution: "Author",
+      });
+    });
+
+    const publisherObj: Publisher[] = [
+      {
+        fullName: publisher,
+        publishYear: parseInt(publicationYear, 10),
+      },
+    ];
+
+    const formatList: BookFormat[] = [
+      {
+        format,
+        price: price.toString(),
+        isbn,
+      },
+    ];
+
+    const newBook: Book = {
+      title,
+      coverImage,
+      titlePrefix: prefix,
+      seriesOrder,
+      illustrator: illustrators,
+      translator: translators,
+      formats: formatList,
+      minAge,
+      maxAge,
+      authors: authorObjs,
+      publishers: publisherObj,
+      seriesName,
+    };
+    handleBooksAdded([...booksAdded, newBook]);
+    onClose();
+  };
+
+  return (
+    <Modal onClose={onClose} isOpen={isOpen}>
+      <ModalOverlay />
+      <ModalContent maxW="70%" padding="10px">
+        <ModalHeader>Add a new book</ModalHeader>
+        <ModalBody>
+          <Grid templateColumns="repeat(2, 1fr)" gap={10}>
+            <GridItem>
+              <Stack spacing={4}>
+                <Stack w="100%" spacing={5} isInline>
+                  <AddStringInput
+                    id="titlePrefix"
+                    label="Prefix"
+                    name="title-prefix"
+                    required
+                    maxWidth="50%"
+                    inputField={prefix}
+                    setInputField={setPrefix}
+                  />
+                  <AddStringInput
+                    id="title"
+                    label="Title"
+                    name="title"
+                    required
+                    inputField={title}
+                    setInputField={setTitle}
+                  />
+                </Stack>
+                <Stack align="center" spacing={5} isInline>
+                  <AddStringInput
+                    id="seriesName"
+                    label="Series"
+                    name="series-name"
+                    required={false}
+                    inputField={seriesName}
+                    setInputField={setSeriesName}
+                  />
+                  <AddStringInput
+                    id="seriesOrder"
+                    label="Book number"
+                    name="series-order"
+                    required={false}
+                    maxWidth="50%"
+                    inputField={seriesOrder}
+                    setInputField={setSeriesOrder}
+                  />
+                </Stack>
+                <AddStringInputList
```

Comment:
It seems like when I try to add 2 authors, I get this error on the browser console:
![image](https://user-images.githubusercontent.com/57159970/157335462-677a0a29-1c58-4491-b640-e767e9ee4850.png)
Could we look into this so the console is clean?

Reply:
seems to only be happening the first time you try to add 2 authors. Also only happening for authors not any other list fields.

Reply:
Good catch, found it in `theme/index.tsx`. We had a css property `_hover` instead of `hover` for the Button variant "remove".